sjb/config/test_cases/test_pull_request_origin_aggregated_logging_ansible.yml

```diff
@@ -17,7 +17,11 @@ extensions:
       title: "build an openshift-ansible release"
       repository: "openshift-ansible"
       script: |-
-        git checkout "${PULL_REFS%%:*}"
+        if [[ "${PULL_REFS%%:*}" == es5.x ]]; then
+          git checkout "${OPENSHIFT_ANSIBLE_TARGET_BRANCH}"
```

Comment:
I don't think this will be set to anything other than `master`.

Reply:
that's fine, before I wasn't using an if else block, so it was set to `OPENSHIFT_ANSIBLE_TARGET_BRANCH` all the time. If we're running ci for the es5.x branch we should use the ansible target branch (which is fine to be `master` always)